rest-api-spec/src/main/resources/rest-api-spec/api/security.search_user_profile.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Should this be plural, that is `search_user_profiles`? We tend to be use plurals for API over multiple entities, from what I see.

Reply:
Changed to plural.